### DISCUSSION TITLE
How can I use resnet 18?

### DISCUSSION THREAD

**Discussion by debparth** [facebookresearch/detectron2#2877] [General]
I'm a newbie in detectron2. I saw the different models in model zoo, but I want to use resnet 18 for training on custom dataset (with segmentation) using detectron2. How can I use it?

**Comment by shehroz010** [comment]
You can do that by creating your own backbone on ResNet-18 and then registering it. 
`from detectron2.modeling import BACKBONE_REGISTRY, Backbone`

Link:
[https://detectron2.readthedocs.io/en/latest/tutorials/write-models.html](url)
